### DISCUSSION TITLE
@Blacklisted long URL.

### DISCUSSION THREAD

**Discussion by Profy13** [jaykali/maskphish#22] [Q&A] [answered]
![Screenshot_20210302-105432_Termux](https://user-images.githubusercontent.com/39966758/109618357-7f632d80-7b48-11eb-9fb1-81c625552ae9.png)

Hello.  I do everything according to the instructions, but I always end up with a @Blacklisted long URL. Thanks.

**Comment by jaykali** [comment, marked as answer]
You are using older version of tool. Kindly delete it, and download the updated tool. Thanks.

**Reply by Profy13** [reply]
Thanks!  It would be great if you specified the version exactly when using your script in the terminal, so that you can see which one you are using right now, thanks again!

**Reply by jaykali** [reply]
use the latest version.

**Comment by jaykali** [comment]
Just follow following commands:
```
rm -rf maskphish
git clone https://github.com/jaykali/maskphish
cd maskphish
bash maskphish.sh
```

**Comment by Shubh-singhh** [comment]
Still getting same problem
It was alright when i was masking before but now when i really needed it
Now it's not working for a certain URLs
Btw it was working on them before but not after a while

**Reply by jaykali** [reply]
It's perfectly working here. I suggest you to use VPN.

**Reply by Shubh-singhh** [reply]
I was using with/without tor

**Comment by Kavinsky29** [comment]
I have the latest version and running behind a VPN. Tried many URLs and SE words and keep getting the same blacklisted error.

**Comment by juanzu** [comment]
I have the same error and I have the latest version. I tried with several words but it didn't work
